software/kwallet/en.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "KDE Wallet and SSH keys"
+lastmod = "2019-03-16T14:50:00+01:00"
++++
+# Using KDE Wallet to manage SSH key passphrases on Solus KDE Plasma Desktop
```

Comment:
This should really be the same title as the doc itself, so "KDE Wallet and SSH keys"